examples/gallery/build-tools/constants.js

```diff
@@ -8,7 +8,6 @@ const IMAGE_DIR = 'images';
 const MAPBOX_TOKEN = process.env.MapboxAccessToken;
 if (!MAPBOX_TOKEN) {
   console.log('Missing environment variable: MapboxAccessToken');
-  process.exit(1);
```

Comment:
If the token is required by some example this line shouldn't be removed

Reply:
Fixed: e172c00